### DISCUSSION TITLE
Change default realm - admin console

### DISCUSSION THREAD

**Discussion by old-syniex** [keycloak/keycloak#12915] [Q&A]
Is there a way to change the default realm when entering admin console.
I created a new realm and disabled master realm, but could not find how to change it.

Thanks

**Comment by ssilvert** [comment]
http://localhost:8080/admin/master/console/#/realms/myrealm

**Reply by old-syniex** [reply]
This won't change the default link of the admin console.

**Reply by ssilvert** [reply]
What default link are you referring to?

**Reply by old-syniex** [reply]
![keyclock-welcome-page.jpg](https://user-images.githubusercontent.com/10838928/177632295-5b9e216e-b904-4e7c-b07d-4222ce530f92.jpg)

**Reply by ssilvert** [reply]
Oh, that just comes from a Freemarker template. You need to hack the index.ftl in the Welcome theme.
`<h3><a href="${adminUrl}"><img src="welcome-content/user.png">Administration Console <i class="fa fa-angle-right link" aria-hidden="true"></i></a></h3>`

You could also change the  value of `adminUrl`, but it sounds like that's not what you are really wanting to do.  So I would just hard-code it to the realm you want using the URL from my previous post as an example.
